Automattic/wp-calypso
# Plugins: Cleanup unused plugin state

#### Changes proposed in this Pull Request

As I was working on reviewing #61803, I noticed there's plenty of unused plugins state, most of which we're bundling with the production code. This PR aims to clean it up to remove some dead weight.

#### Testing instructions

* Verify none of the removed code is in use.
* Verify Calypso builds properly.
* Verify all tests pass.
* Smoke test `/plugins` and its various views.
